interrupts.c

```diff
@@ -4,28 +4,6 @@
 
 extern const char _ebase[];
 
-void intr_init() {
```

Comment:
Please try to avoid moving code around the file. This distracts a reader! I personally learned that it's best to keep PR as small as possible.